save-demo/src/main/kotlin/com/saveourtool/save/demo/entity/Tool.kt

```diff
@@ -0,0 +1,30 @@
+package com.saveourtool.save.demo.entity
+
+import com.saveourtool.save.demo.storage.ToolKey
+import com.saveourtool.save.spring.entity.BaseEntity
+import javax.persistence.*
+
+/**
+ * @property gitRepo
+ * @property snapshot
+ */
+@Entity
+@Table(name = "tool")
+class Tool(
```

Comment:
Tool relates only to github, maybe relates to our internal storage (file-based in first phase and S3 in second phase).
A service to download tool from github can be added as a separate service and as only one from several (manual upload for example)